### PR TITLE
Changes from mi123

### PR DESCRIPTION
**Pull request by mipatil123**

### CODE REVIEW

File: git.js [hunk 1]
```diff
@@ -5,3 +5,4 @@ console.log("Two files");
 
 
 console.log("1st branch");
+console.log("I am from 123");
```
Comment (meanagha) [inline git.js]: I like your work but i want to make some changes.

Comment (mipatil123) [reply git.js]: Which change

Comment (meanagha) [reply git.js]: i ma from mipatil123 kar as

Comment (mipatil123) [reply git.js]: ok

**Pull request merged**